{
    "config": {
        "abort": {
            "already_configured": "Hizmet zaten yap\u0131land\u0131r\u0131lm\u0131\u015f",
            "reauth_successful": "Yeniden kimlik do\u011frulama ba\u015far\u0131l\u0131 oldu"
        },
        "error": {
            "cannot_connect": "Ba\u011flanma hatas\u0131",
            "invalid_auth": "Ge\u00e7ersiz kimlik do\u011frulama",
            "unknown": "Beklenmeyen hata"
        },
        "step": {
            "reauth_confirm": {
                "data": {
                    "api_key": "API Anahtar\u0131"
                },
                "description": "L\u00fctfen DataHub API anahtar\u0131n\u0131z\u0131 yeniden girin. Hala eski bir Datapoint API anahtar\u0131 kullan\u0131yorsan\u0131z, \u015fimdi DataHub API'ye kaydolman\u0131z gerekir, ayr\u0131nt\u0131lar i\u00e7in [documentation]( {docs_url} )'a bak\u0131n.",
                "title": "DataHub API ile yeniden kimlik do\u011frulamas\u0131 yap\u0131n"
            },
            "user": {
                "data": {
                    "api_key": "API Anahtar\u0131",
                    "latitude": "Enlem",
                    "longitude": "Boylam"
                },
                "title": "\u0130ngiltere Met Office'e ba\u011flan\u0131n"
            }
        }
    }
}